common/api/imodeljs-common.api.md

```diff
@@ -7893,6 +7893,13 @@ export interface TextStringProps {
     widthFactor?: number;
 }
 
+// @public
+export interface TextureData {
+    bytes: Uint8Array;

```

Comment:
This is a JPEG or PNG, correct? You happen to know which at your call site, right - but shouldn't the result of queryTextureImage include the format?

Reply:
Seems like a good idea, added it.